tools/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
+# Retags all docker images from latest Environments build
+# tools/scripts/update-docker-tags.sh NEW_VERSION [--release]
+
+if [ "$#" -lt 1 ] || [ "$#" -gt 1 ] || [ "$#" -eq 2 ] && [ "$2" != "--release" ]; then
+	echo "usage: $0 NEW_VERSION [--release]" >&2
+	exit 1
+fi
+
+# check for dirty changes
+if [[ -n "$(git status --porcelain)" ]]; then
+	echo "untracked or dirty files are not allowed, cleanup before running update-docker-tags.sh"
+	exit 1
+fi
+
+# parse tag args
+export OLD_TAG=$(cat tools/scripts/environments-target.txt)
+export NEW_TAG="$1"
+
+# get list of images to replace via OLD_TAG in bumpenvs.yaml
+export IMAGES=$(grep -oP "(?<=new: ).*(?=,)" tools/scripts/bumpenvs.yaml | grep $OLD_TAG)
```

Comment:
```suggestion
export IMAGES=$(grep -oP "(?<=new: ).*(?=,)" tools/scripts/bumpenvs.yaml | grep -F $OLD_TAG)
```
Since the tag contains periods which should be matched as periods, treat the pattern as a fixed string.